test/apps/aggregator/histogram-layer.ts

```diff
@@ -0,0 +1,170 @@
+import {
+  CPUAggregator,
+  WebGLAggregator,
+  AggregationOperation,
+  _AggregationLayer
+} from '@deck.gl/aggregation-layers';
+import {GridCellLayer} from '@deck.gl/layers';
+import type {Accessor, Color, DefaultProps, UpdateParameters} from '@deck.gl/core';
+import {Matrix4} from '@math.gl/core';
+
+export type HistogramLayerProps<DataT = unknown> = {
+  data: DataT[];
+  getPosition?: Accessor<DataT, number>;
+  getWeight?: Accessor<DataT, number>;
+  binSize?: number;
+  aggregation?: AggregationOperation;
+  gpuAggregation?: boolean;
+  fillColor?: Color;
+  heightScale?: number;
+};
+
+const defaultProps: DefaultProps<HistogramLayerProps> = {
+  getPosition: {type: 'accessor', value: (d: any) => d.bin},
+  getWeight: {type: 'accessor', value: 1},
+  binSize: 1,
+  aggregation: 'SUM',
+  gpuAggregation: false,
+  fillColor: {type: 'color', value: [0, 0, 0, 255]},
+  heightScale: {type: 'number', min: 0, value: 1}
+};
+
+export class HistogramLayer<DataT = unknown> extends _AggregationLayer<
```

Comment:
Should we not promote `_AggregationLayer` to `AggregationLayer`?

Reply:
There are still a bunch of discussions around the API. I would give it a version to mature